eng/pipelines/runtime-android.yml

```diff
@@ -1,9 +1,21 @@
-trigger:
-- none
+# This is a wrapper yml for `runtime-extra-platforms-android.yml`, which
+# has all the Android jobs. This file is essentially so we can have point
+# the pipeline in azdo UI to this, and thus avoid any scheduled triggers
 
-pool:
-  vmImage: 'ubuntu-latest'
+trigger: none
 
-steps:
-- script: echo Hello, world!
-  displayName: 'Run a one-line script'
+variables:
+  - template: /eng/pipelines/common/variables.yml
+
+jobs:
+
+#
+# Evaluate paths
+#
+- template: /eng/pipelines/common/evaluate-default-paths.yml
```

Comment:
These top level pipelines when run manually, will always run their jobs. Is this needed at all then?